redis/3-highly-available/usr/local/share/redis_functions.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+function get_existing_master()
+{
+  redis-cli -h redis-sentinel -p 26379 --csv SENTINEL get-master-addr-by-name mymaster | tr ',' ' ' | cut -d' ' -f1 | cut -d'"' -f2
```

Comment:
maybe you can abstract `redis-cli -h redis-sentinel -p 26379` in a function?

Reply:
Done!